lib/ui/page/home/page/chat/controller.dart

```diff
@@ -309,16 +309,20 @@ class ChatController extends GetxController {
       onSubmit: () async {
         if (send.forwarding.value) {
           if (send.replied.isNotEmpty) {
-            bool? result = await ChatForwardView.show(
-              router.context!,
-              id,
-              send.replied.map((e) => ChatItemQuoteInput(item: e)).toList(),
-              text: send.field.text,
-              attachments: send.attachments.map((e) => e.value).toList(),
-            );
-
-            if (result == true) {
-              send.clear();
+            if (send.replied.any((e) => e is ChatCall)) {
+              MessagePopup.error('err_call_forwarding'.l10n);
```

Comment:
```suggestion
              await MessagePopup.error('err_call_forwarding'.l10n);
```

Лучше `await`ить то, что можно. Чтобы не было потом плавающих футур.

Reply:
@SleepySquash У нас в коде 100+ мест где мы не `await`им `MessagePopup.error`, мне во все места `await` дописать?

Reply:
@krida2000, уф. Вопрос хороший. Я думаю, не стоит тогда.